blocks/library/freeform/old-editor.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * WordPress dependencies
+ */
+import { Component } from 'element';
+import { __ } from 'i18n';
+
+export default class OldEditor extends Component {
+	constructor( props ) {
+		super( props );
+		this.initialize = this.initialize.bind( this );
+		this.onSetup = this.onSetup.bind( this );
+	}
+
+	componentDidMount() {
+		const { baseURL, suffix } = window.wpEditorL10n.tinymce;
+
+		window.tinymce.EditorManager.overrideDefaults( {
+			base_url: baseURL,
+			suffix,
+		} );
+
+		if ( document.readyState === 'complete' ) {
+			this.initialize();
+		} else {
+			window.addEventListener( 'DOMContentLoaded', this.initialize );
+		}
+	}
+
+	componentWillUnmount() {
+		window.addEventListener( 'DOMContentLoaded', this.initialize );
+		wp.oldEditor.remove( this.props.id );
+	}
+
+	componentDidUpdate( prevProps ) {
+		const { id, attributes: { content } } = this.props;
+
+		if ( prevProps.attributes.content !== content ) {
+			window.tinymce.get( id ).setContent( content || '' );
+		}
+	}
+
+	initialize() {
+		const { id } = this.props;
+		const { settings } = window.wpEditorL10n.tinymce;
+
+		wp.oldEditor.initialize( id, {
+			tinymce: {
+				...settings,
+				inline: true,
+				content_css: false,
+				fixed_toolbar_container: '#' + id + '-toolbar',
+				setup: this.onSetup,
+			},
+		} );
+	}
+
+	onSetup( editor ) {
+		const { attributes: { content }, setAttributes } = this.props;
+		const { ref } = this;
+		const initialContent = window.switchEditors.wpautop( content || '' );
+
+		editor.on( 'loadContent', () => editor.setContent( initialContent ) );
+
+		editor.on( 'blur', () => {
+			setAttributes( {
+				content: editor.getContent(),
+			} );
+		} );
+
+		editor.addButton( 'kitchensink', {
+			tooltip: __( 'More' ),
+			icon: 'dashicon dashicons-editor-kitchensink',
+			onClick: function() {
+				const button = this;
+				const active = ! button.active();
+
+				button.active( active );
+				editor.dom.toggleClass( ref, 'has-advanced-toolbar', active );
+			},
+		} );
+
+		/* eslint-disable */
+
+		// Re-register WP_More as it doesn't work with inline mode.
+		// This should be fixed in core.
+		// See wp-includes/js/tinymce/plugins/wordpress/plugin.js
+		// Swaps node.nodeName === 'BODY' to node === editor.getBody()
+		editor.on( 'init', () => editor.addCommand( 'WP_More', function( tag ) {
+			var parent, html, title,
+				classname = 'wp-more-tag',
+				dom = editor.dom,
+				node = editor.selection.getNode(),
+				rootNode = editor.getBody();
+
+			tag = tag || 'more';
+			classname += ' mce-wp-' + tag;
+			title = tag === 'more' ? 'Read more...' : 'Next page';
```

Comment:
Should these be translated? Also ellipsis character `…` as more semantically meaningful than three dots.